docs/workflows/logic/geofence.md

```diff
@@ -6,20 +6,23 @@ The geofence node allows a workflow to branch based upon the distance between tw
 
-The geofence node has 4 distinct configuration sections - the coordinates to check (the "input" coordinates), the center point coordinates, the center point radius, and an optional payload destination for the calculated distance.
+The geofence node has 4 distinct configuration sections - the coordinates to check (the "input" coordinates), the center point coordinates, the center point radius, and an optional [payload path](/workflows/accessing-payload-data/#payload-paths) for the calculated distance.
 
-When the geofence node runs, it calculates the distance between the input and center point coordinates, and if that distance is less than or equal to the defined center point radius, the `true` route on the right out of the node is taken.  If that distance is greater than the center point radius, the `false` route on the left out of the node is taken.  If the distance between the two points cannot be calculated for some reason, the `false` route on the left out of the node is taken.
+When the geofence node runs, it calculates the distance between the input and center point coordinates, and if that distance is less than or equal to the defined center point radius, the `true` route on the right out of the node is taken. If that distance is greater than the center point radius, the `false` route on the left out of the node is taken. If the distance between the two points cannot be calculated for some reason, the `false` route on the left out of the node is taken.
 
 ![Geofence Node Example](/images/workflows/logic/geofence-node-example.png "Geofence Node Example")
 
-In the above example, there are actually two different geofences.  The first checks against a center point position that represents "Home".  If that is true (the route to the right), it records that payload as being "At Home".  If that is false (the route to the left), it moves on to the second geofence, which checks against a center point position that represents "Work".  If that is true (the route to the right), it records that payload as being "At Work", but if that is false (the route to the left), it records that payload as being "Other".
+In the above example, there are actually two different geofences. The first checks against a center point position that represents "Home". If that is true (the route to the right), it records that payload as being "At Home". If that is false (the route to the left), it moves on to the second geofence, which checks against a center point position that represents "Work". If that is true (the route to the right), it records that payload as being "At Work", but if that is false (the route to the left), it records that payload as being "Other".
 
 ### Input Coordinate Configuration
 
 ![Geofence Node Input Config](/images/workflows/logic/geofence-node-input-config.png "Geofence Node Input Config")
 
 The input coordinates can be defined four different ways:
 
+*   [String template](/workflows/accessing-payload-data/#string-templates)<br/>
```

Comment:
I wouldn't say this is a format for defining gps, all 4 of the listed formats can be input using string templates.  Probably this just belongs at the end of the section, saying you can use templates on this field